packages/shared/src/components/checklist/ChecklistCard.tsx

```diff
@@ -29,9 +30,10 @@ const ChecklistCard = ({
           <p className="mb-1 font-bold text-white typo-body">{title}</p>
           <p className="text-white typo-callout">{description}</p>
           {typeof onRequestClose === 'function' && (
-            <CloseButton
-              buttonSize={ButtonSize.Small}
-              className="top-3 right-3 text-white border-white !absolute btn-secondary"
+            <ModalClose
+              size={ButtonSize.Small}
+              variant={ButtonVariant.Secondary}
+              className="top-3 right-3"
```

Comment:
We should just be careful as `modalClose` already has `right-2` by default.
Meaning we could end up with both classes and depending on tailwind version the order changes it executes them.

Reply:
See [1b2d07a](https://github.com/dailydotdev/apps/pull/2435/commits/1b2d07a83d32f72e5b63a607d860a921955545b3)